src/styles/heading.scss

```diff
@@ -1,10 +1,31 @@
 @import "./_variables";
 
-h1,
-h2,
-h3,
-h4,
-h5,
+h1 {
+    font-family: $heading-font;
```

Comment:
group the headers together into one declaration for easy editing e.g.

h1,
h2,
h3,
h4,
h5 {
font-family:$heading-font;
}

Reply:
headers have been grouped. h1 & h2 have specific size and letter spacing. the other headers can be generic.